examples/leds/leds_main.c

```diff
@@ -170,16 +197,25 @@ static int led_daemon(int argc, char *argv[])
           goto errout_with_fd;
         }
 
-      usleep(500*1000L);
+      usleep(500 * 1000L);
+    }
+
+  /* treats signal termination of the task */
+
+  if (g_led_daemon_started == false)
+    {
+      /* task terminated by a SIGTERM */
+
+      exit(EXIT_SUCCESS);
```

Comment:
The idea was to force the execution of a exit(EXIT_SUCCESS) instead of exiting with error code (last line if normal flow).
POSIX exit() terminates the task, closes all fds, etc. I think it is safer.
I can remove the if(...) { ... }, but the not the exit(EXIT_SUCCESS) given it was correctly terminated by a SIGTERM at that point.

Reply:
Ok, just remove the if() will be fine